packages/web-components/src/components/text-input/text-input.scss

```
@@ -57,3 +58,7 @@ $css--plex: true !default;
     inset-inline-end: $spacing-08;
   }
 }
+
+:host([ai-label]) .#{$prefix}--text-input__field-wrapper--slug {
```

```suggestion
:host(#{$prefix}-text-input[ai-label]) .#{$prefix}--text-input__field-wrapper--slug {
```

We should include the component name in the `host` selector just for consistency. In other components, there is a shared stylesheet with multiple sub-components - so this makes it easier to find which styles belong to which component.